orion/pkg/domains/user.go

```diff
@@ -13,14 +13,14 @@ type User struct {
 	Id         uint
 	CreatedAt  time.Time    `db:"created_at"`
 	UpdatedAt  time.Time    `db:"updated_at"`
-	DeletedAt  sql.NullTime `db:"deleted_at"`
+	DeletedAt  sql.NullTime `db:"deleted_at" json:"-"`
```

Comment:
this part will signal to not include this field in the resulting JSON. So web clients will not be able to see `DeletedAt`